docs/codeql/reusables/threat-model-description.rst

```diff
@@ -4,7 +4,16 @@ A threat model is a named class of dataflow sources that can be enabled or disab
 
 The ``kind`` property of the ``sourceModel`` determines which threat model a source is associated with. There are two main categories:
 
-- ``remote`` which represents requests and responses from the network.
-- ``local`` which represents data from local files (``file``), command-line arguments (``commandargs``), database reads (``database``), and environment variables(``environment``).
+- ``remote`` which represents requests (``request``) and responses (``response``) from the network.
+- ``local`` which represents data from local files (``file``), command-line arguments (``commandargs``), database reads (``database``), environment variables(``environment``) and Windows registry values ("windows-registry").
+
+Note that subcategories can be turned included or excluded separately, so you can specify ``local`` without ``database``, or just ``commandargs`` and ``environment`` without the rest of ``local``.
+
+The less commonly used categories are:
+
+- ``android`` which represents reads from external files in Android (``android-external-storage-dir``) and parameter of an entry-point method declared in a ``ContentProvider`` class (``contentprovider``).
```

Comment:
I'm assuming this is kotlin only (though maybe Java as well?).

```suggestion
- ``android`` which represents reads from external files in Android (``android-external-storage-dir``) and parameter of an entry-point method declared in a ``ContentProvider`` class (``contentprovider``). Currently only used by Kotlin.
```

Reply:
It's kind of hard to tell, but at least one of the tests for this is written in java. I'll put "Java/Kotlin".